### ISSUE TITLE
Clarify exposure type field

### ISSUE THREAD

**Issue by stufraser1** [GFDRR/rdl-data#22]
this field not clear (mainly when 'mixed' is the value - more intuitive when 'Residential', 'Commercial' value given) 
PC test: 'What does Exposure type mean?'

**Comment by MamadioCMCC** [comment]
Agree, exposure type refers to _occupancy_enum_ which is intuitive to understand apart when it is "mixed".

  ```
Residential
  Commercial
  Industrial
  Infrastructure
  Healthcare
  Educational
  Government
  Crop
  Livestock
  Forestry
  Mixed
```

"Mixed _**use**_" or "Mixed _**occupancy**_" maybe explains better we are talking of land use or building occupancy.

**Comment by stufraser1** [comment]
Agree to update to "Mixed occupancy" @MamadioCMCC please action this

**Comment by stufraser1** [comment]
If we are able to upload more data to AWS DB, then yes I think it is wise to update the schema with this and other changes before uploading data there. But this is dependent on import workflow I think. If the workflow still hampers upload, and we plan to upload data in the revised system with easier import workflow, then lets just make the changes in the alchemy version

**Comment by ConnectedSystems** [comment]
Changes have been made to the schema implementation but I'll leave this issue open until questions around the DB are resolved.
